php/aplus-app/config.yaml

```
@@ -10,6 +10,7 @@ framework:
   build_deps:
     - libsodium-dev
     - memcached
+    - libz-dev
     - libmemcached-tools
     - libmemcached-dev
```

It looks fine, try adding a `libssl-dev`

```suggestion
    - libmemcached-dev
    - libssl-dev
```